app/src/main/java/de/westnordost/streetcomplete/quests/barrier_bicycle_barrier_type/AddBicycleBarrierTypeForm.kt

```diff
@@ -0,0 +1,23 @@
+package de.westnordost.streetcomplete.quests.barrier_bicycle_barrier_type
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.quests.AImageListQuestAnswerFragment
+import de.westnordost.streetcomplete.quests.barrier_bicycle_barrier_type.BicycleBarrierType.*
+import de.westnordost.streetcomplete.view.image_select.Item
+
+class AddBicycleBarrierTypeForm : AImageListQuestAnswerFragment<BicycleBarrierType, BicycleBarrierType>() {
+
+    override val items = listOf(
+        Item(SINGLE, R.drawable.barrier_bicycle_barrier_single, R.string.quest_barrier_bicycle_type_single),
+        Item(DOUBLE, R.drawable.barrier_bicycle_barrier_double, R.string.quest_barrier_bicycle_type_double),
+        Item(TRIPLE, R.drawable.barrier_bicycle_barrier_triple, R.string.quest_barrier_bicycle_type_multiple),
+        Item(DIAGONAL, R.drawable.barrier_bicycle_barrier_diagonal, R.string.quest_barrier_bicycle_type_diagonal),
+        Item(TILTED, R.drawable.barrier_bicycle_barrier_tilted, R.string.quest_barrier_bicycle_type_tilted),
```

Comment:
I think that the images of `single`, `triple` and `diagonal` are a bit faint compared to `double` and `tilted`. Maybe adjust the contrast a bit for them?

Reply:
went through tweaking round - do you think it is OK now?
![screen12](https://user-images.githubusercontent.com/899988/140479156-2964a3b5-2a6f-4de9-96ae-a32215382bd8.png)

Reply:
It's a lot better. IMO the contrast of `diagonal` could be increased a little more, and the contrast of `third` could still be increased by a lot – but maybe that distorts colors too much there, so a different image would be needed. In any case, this is not merge-blocking.